### PR TITLE
fix: update azure time out logic

### PR DESCRIPTION
**Pull request by xiaolang124**
1. Throw timeout error when azure login does not success.

### CODE REVIEW

**Comment by wenytang-ms** [comment]
Lint-PR check fail, just rebase or merge latest dev code

**Review by swatDong — COMMENTED** [review]
Is this only to change system error to user error? Any UX requirement?

File: packages/vscode-extension/src/commonlib/azureLogin.ts [hunk 1]
```diff
@@ -161,6 +161,14 @@ export class AzureAccountManager extends login implements AzureAccountProvider {
       [TelemetryProperty.AccountType]: AccountType.Azure,
     });
     await vscode.commands.executeCommand("azure-account.login");
+    const azureAccount = this.getAzureAccount();
+    if (azureAccount.status != loggedIn) {
```
Comment (a1exwang) [inline packages/vscode-extension/src/commonlib/azureLogin.ts:165]: Does CLI also have this bug?

**Comment by xiaolang124** [comment]
> Is this only to change system error to user error? Any UX requirement?

This is only fix for system error. No UX requirement.
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/10957287

File: packages/vscode-extension/src/commonlib/azureLogin.ts (hunk 1 above)
Comment (xiaolang124) [reply packages/vscode-extension/src/commonlib/azureLogin.ts:165]: CLI Azure login is handled by codeFlow and will throw error if timeout. VsCode azure-account-extension will catch this error and cause this bug.

**Review by swatDong — APPROVED** [review]

**Comment by xiaolang124** [comment]
> Lint-PR check fail, just rebase or merge latest dev code

Thanks for the advice.

**Pull request merged**